### ISSUE TITLE
BUG brainrender scene fails to display (apart from inset)

### ISSUE THREAD

**Issue by alessandrofelder** [brainglobe/brainrender#240]
**Describe the bug**
Running a simple test script (or any of the examples) doesn't display and just stays black.

**To Reproduce**
* on latest `brainrender` cloned from GitHub at commit a14ead80c1dbc75f20a145a49394dc467c4f7bf1 and `pip install -e .`d
* on Ubuntu 22.04.2 LTS
* vedo --info gives
```
vedo version      : 2021.0.5   https://vedo.embl.es
vtk version       : 9.2.6
python version    : 3.8.17 | packaged by conda-forge | (default, Jun 16 2023, 07:06:00) [GCC 11.4.0]
python interpreter: /home/alessandro/mambaforge/envs/brainrender-spyder1/bin/python3.8
vedo installation : /home/alessandro/mambaforge/envs/brainrender-spyder1/lib/python3.8/site-packages/vedo
system            : Linux 5.19.0-50-generic posix x86_64
k3d version       : 2.7.4
```
* conda env with Python 3.8

Then tell us the steps to reproduce the behavior:
* run the below script (or any of the examples AFAICT) 
```python
import brainrender
scene=brainrender.Scene(atlas_name="allen_mouse_25um")
scene.render()
```
* see a black, blank scene in the brainrender window (but with the inset on the lower right)

**Expected behavior**
A reasonable looking 3d scene with the allen mouse atlas (25 um)  in the brainrender style should appear

**Screenshots**
![Screenshot from 2023-08-04 16-10-56](https://github.com/brainglobe/brainrender/assets/10500965/9327dc82-fb1a-492e-ae90-80b15557939d)
See also [discussion on image.sc](https://forum.image.sc/t/how-to-display-own-dataset-in-brainrender/84547)

**Comment by alessandrofelder** [comment]
Setting `vsetting.use_fxaa = False` in [settings.py](https://github.com/brainglobe/brainrender/blob/69a2aeabd3fa1b97357fffdbecd7844081ddbed7/brainrender/settings.py#L24) fixes this for me in a `[dev]` installation of the current `mega-fix` branch.

Initial investigation suggests this is an `nvidia` anti-aliasing setting, so it looks like this is the result of an interplay between `vedo` and me not having my GPU set up correctly?

**Comment by adamltyson** [comment]
> Initial investigation suggests this is an nvidia anti-aliasing setting, so it looks like this is the result of an interplay between vedo and me not having my GPU set up correctly?

Can you reproduce on your macbook?

**Comment by marcomusy** [comment]
I think these 2 flags should be set to false by default as they are machine hence user-dependent.

**Comment by alessandrofelder** [comment]
Fixed by #283
